README.md

```diff
@@ -123,6 +123,7 @@ Spiderable.debug = true
  - Create template which you prefer to return, when page is not found
  - Set `Spiderable.customQuery`
  - Set iron router's `notFoundTemplate`
+ - Include a comment '<!-- response:status-code=404 -->' on your template. This way, we can ensure spiderable sends a 404 status code in the response headers
```

Comment:
It does appear to be the case.
I don't see a big risk to these changes, since it only adds a little optional functionality.
If there are no objections, I will tag and publish to Atmosphere tomorrow.